SU-HSDO/suhumsci
# STN-348: Add Font Family Selection for Traditional Theme

# READY FOR REVIEW

## Summary
Add the ability to swap between Sans-Serif and Serif font families for headings and page titles for the Traditional theme.
Jira Card: [STN-348](https://sparkbox.atlassian.net/browse/STN-348)

## Urgency
medium

## Steps to Test
1. npm run test
2. To change the font family and test this card:
- Ensure you are on the Traditional theme as this setting only applies to it.
- Go to Appearance > HumSci Traditional > Settings > Scroll down to "Font Family Settings" and choose 'Sans-Serif'.
3. If you inspect the page as it loads you should see, `hb-font-sans-serif` class being applied to the <body> tag.
![image](https://user-images.githubusercontent.com/9355698/92279496-69d8b800-eec5-11ea-8e1b-845113fbf0a1.png)

4. All H1, H2 and H3 headings and page titles should now be the Sans Serif font, rather than the default Serif font family.
![image](https://user-images.githubusercontent.com/9355698/92279610-aad0cc80-eec5-11ea-93f7-53cd759079e6.png)


## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
